electrum/constants.py

```diff
@@ -152,17 +152,53 @@ class BitcoinSimnet(BitcoinTestnet):
     LN_DNS_SEEDS = []
 
 
+class ElectricCashMainnet(BitcoinMainnet):
+
+    WIF_PREFIX = 0x80
+    ADDRTYPE_P2PKH = 33
+    ADDRTYPE_P2SH = 87
+    SEGWIT_HRP = "elcash"
+    GENESIS = "0000000028ce26975b32feda3d75ac3fe10372f75062366cfba4e934dcc6a48b"
+    DEFAULT_SERVERS = read_json('servers_elcash.json', {})
```

Comment:
I think we could add these new config files, even if they will be empty for now.

Reply:
done